src/cli/cmd_clean.py

```diff
@@ -177,6 +233,11 @@ def clean(
         )
         if not should_delete:
             sys.exit()
+
+    # save file verification status to a csv file for idw project
+    if profile_name == "idw":
+        logger.info("Saving files verification status into a csv file.")
+        _save_data_status(source_data_path, verified_dfs, unverified_dfs)
```

Comment:
I think it would be useful to have a separate command for the reporting functionality. The `ids clean` command mainly deletes files, whereas this functionality creates a report. To prevent accidental deleting of files it's better for the CLI user to run it as a separate command...

Reply:
I separated the cmd_report from cmd_clean just in case it will interrupt other workflows.